crud.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorBaseCreate, BookBaseCreate
+
+
+def get_authors(db: Session):
+    return db.query(models.DBAuthor).all()
```

Comment:
The `get_authors` function should support pagination by accepting `skip` and `limit` parameters to retrieve a subset of authors. Consider adding these parameters and using them in the query.